db/migrate/20210601115051_remove_maintenance_task_entries.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMaintenanceTaskEntries < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :maintenance_task
```

Comment:
drop_table is not reversible without attrs defined in a block. 

http://joshfrankel.me/blog/ensure-dropping-a-database-table-is-reversible/